File: Suit.java
public class Suit extends Costume{
    private String type;

    Suit(String color, int size, int price, String type) {
        super(color, size, price);
        this.type = type;
    }

    @Override
    public void print() {
        super.print();
        System.out.println("“ип: " + type);
    }

    @Override
    public int getPrice() {
        return (int) (price * 1.01);
    }
}

File: c_sharp/Suit.cs
using System;

public class Suit : Costume
{
    private String type;

    public Suit(String color, int size, int price, String type) : base (color, size, price) {
        this.type = type;
    }


    public override void print()
    {
        base.print();
        Console.WriteLine("Тип: " + type);
    }
}